manual.asc

```diff
@@ -562,6 +562,23 @@ This is not officially supported, but `.anki2` files can manually be imported if
 * (If applicable) Temporarily log out of your AnkiWeb account: `Settings - AnkiDroid - AnkiWeb account`
 * Open `Settings - Advanced - AnkiDroid Directory` and set the AnkiDroid Directory to the newly created folder.
 
+== Importing Comma-seperated Value files and Text files (.txt, .csv)
+Anki now supports the import of .csv and .txt files. The procedure for it on AnkiDroid is as follows:
+* Click the three dots on the upper right hand corner.
+* Choose Import
+* Choose 'Text file (.txt, .csv)'
+* Choose the file from your folders when prompted and click OK
+* In case of .csv, the seperator will be autoselected as comma, but you can choose a suitable seperator for your .txt file from Tab, Pipe, Semicolon, Colon, Comma and Space
+* If you want HTML to be allowed in the fields, turn the button on.
+* In the 'Import options' section, first pick the notetype from options 'Basic', 'Basic (and reversed card)', 'Basic (optional reversed card)', 'Basic (type in the answer)', 'Cloze' and 'Image Occlusion'.
```

Comment:
This is not a fixed list

Reply:
hi, what should i change here? i'm sorry if it's a very trivial question

Reply:
The list of Note Types which a user has is variable.